apps/sample-app/src/components/TokenFetch.tsx

```diff
@@ -0,0 +1,32 @@
+import { useMsal } from '@azure/msal-react';
+import { Spinner, SpinnerSize } from '@fluentui/react';
+import { authentication } from '@microsoft/teams-js';
+import React from 'react';
+
+import { loginRequest } from './authConfig';
+
+const TokenFetchComponent: React.FC = () => {
+  const { instance, accounts } = useMsal();
+
+  const [accessToken, setAccessToken] = React.useState<string>();
+
+  React.useEffect(() => {
+    const request = {
+      ...loginRequest,
+      account: accounts[0],
+    };
+    instance.acquireTokenSilent(request).then(response => {
+      setAccessToken(response.accessToken);
+      authentication.notifySuccess(response.accessToken);
+    });
+  }, [setAccessToken, accounts, instance]);
+
+  return (
+    <>
+      {!accessToken && <p>Fetching access token...</p> && <Spinner size={SpinnerSize.large} />}
```

Comment:
nit: Patterns like this where you check a boolean and then print UI and then on the next line check the opposite and print different UI can sometimes be a little easier to read and maintain if you write them like this:
```suggestion
      {!accessToken ? <><p>Fetching access token...</p><Spinner size={SpinnerSize.large} /></> : <p>...</p>}
```